client/rpc-spec-v2/Cargo.toml

```
@@ -20,14 +20,21 @@ sc-chain-spec = { version = "4.0.0-dev", path = "../chain-spec" }
 sc-transaction-pool-api = { version = "4.0.0-dev", path = "../transaction-pool/api" }
 sp-core = { version = "7.0.0", path = "../../primitives/core" }
 sp-runtime = { version = "7.0.0", path = "../../primitives/runtime" }
+sc-client-api = { version = "4.0.0-dev", path = "../api" }
 sp-api = { version = "4.0.0-dev", path = "../../primitives/api" }
 sp-blockchain = { version = "4.0.0-dev", path = "../../primitives/blockchain" }
 codec = { package = "parity-scale-codec", version = "3.0.0" }
 thiserror = "1.0"
 serde = "1.0"
+array-bytes = "4.1"
 hex = "0.4"
 futures = "0.3.21"
 
 [dev-dependencies]
 serde_json = "1.0"
-tokio = { version = "1.17.0", features = ["macros"] }
+tokio = { version = "1.17.0", features = ["macros", "full"] }
```

full implies macros :)

```suggestion
tokio = { version = "1.17.0", features = ["full"] }
```